tests/Kiota.Builder.Tests/Extensions/OpenApiUrlTreeNodeExtensionsTests.cs

```diff
@@ -82,5 +82,14 @@ public class OpenApiUrlTreeNodeExtensionsTests {
             Assert.Equal("graph.users.messages", node.Children.First().Value.GetNodeNamespaceFromPath("graph"));
             Assert.Equal("users.messages", node.Children.First().Value.GetNodeNamespaceFromPath(null));
         }
+        [Fact]
+        public void SanitizesAtSign() {
+            var doc = new OpenApiDocument {
+                Paths = new(),
+            };
+            doc.Paths.Add("\\deviceManagement\\microsoft.graph.getRoleScopeTagsByIds(ids=@ids)", new() {});
```

Comment:
Add test with 2 parameters.

Reply:
I'm not sure what this would change? we already have tests for multiple parameters above. This one is just for the at sign replacement

Reply:
Previous response about the regex being global preempts this comment..